assets/test/spec/services/apiServiceSpec.js

```diff
@@ -0,0 +1,294 @@
+"use strict";
+
+describe("APIService", function(){
+  var APIService;
+
+  beforeEach(function(){
+    inject(function(_APIService_){
+      APIService = _APIService_;
+    });
+  });
+
+  describe('#qualifyResource', function() {
```

Comment:
these tests are really verbose... I want a LOT more tests around the function, which means they need to be more concise, like the previous spec tests